actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -92,8 +95,15 @@ class ActionCable.Connection
       @disconnected = false
       @subscriptions.reload()
 
-    close: ->
+    close: (event) ->
       ActionCable.log("WebSocket onclose event")
+      if event.code == 1006
+        ActionCable.log("WebSocket received abnormal close error (" + event.code + "): " + event.reason)
+      
+      if !event.wasClean
+        ActionCable.log("WebSocket says close was unclean. Stopping ConnectionMonitor.")
+        @monitor.stop()
+
```

Comment:
Can we bubble this up to the app in some way so it can react in the UI, e.g. show a prompt to reload the page or auto-reload it?